ui/component/livestreamLayout/view.jsx

```diff
@@ -19,8 +18,10 @@ import classnames from 'classnames';
 import usePersistedState from 'effects/use-persisted-state';
 import { getTipValues } from 'util/livestream';
 import 'scss/component/_swipeable-drawer.scss';
-import './style.scss';
 
+const LivestreamScheduledInfo = lazyImport(() =>
```

Comment:
Can't recall, but does the component need to be wrapped in <Suspense>?

Reply:
I don't think so. The mandatory suspense wrappers usually trigger a lint error when they're required and missing.